### DISCUSSION TITLE
Password protection enabled by default and unable to disable it

### DISCUSSION THREAD

**Discussion by rsumit123** [vercel/community#1830] [Help]
### Summary

My vercel project currently has password protection enabled for preview deployments but the toggle options are disabled and it shows "This feature is available with the [Advanced Deployment Protection](https://vercel.com/docs/concepts/deployments/advanced-deployment-protection) add-on for $150 per month". 

I am unable to understand, if we have not opted for this feature, why are we getting password protection by default. I just need to disable it now, and don't want to pay 150 bucks just for that.

Check the below screenshot.

<img width="924" alt="Screenshot 2023-03-21 at 12 56 41 PM" src="https://user-images.githubusercontent.com/24500659/226544306-58eb81a3-7004-4956-9886-d9a785a6c1f5.png">

**Comment by amyegan** [comment]
Thanks for reaching out. I have a few question to help point us in the right direction when investigating the problem: 
- What do you see when you follow the steps to disable password protection? 
  - https://vercel.com/docs/concepts/deployments/advanced-deployment-protection#disable-advanced-deployment-protection
- Was this project transferred from another account or team? If so, did that team have password protection enabled?

It looks like you also opened a support case with our Customer Success Team. If you prefer to share the info privately there, that works perfectly fine as well 🙂
